Mathlib/Order/Filter/Bases.lean

```diff
@@ -1075,6 +1079,10 @@ theorem HasAntitoneBasis.hasBasis_ge [Preorder ι] [IsDirected ι (· ≤ ·)] {
     ⟨j, trivial, Subset.rfl⟩
 #align filter.has_antitone_basis.has_basis_ge Filter.HasAntitoneBasis.hasBasis_ge
 
+theorem HasAntitoneBasis.isCountablyGenerated [Preorder ι] [Countable ι] {l : Filter α}
```

Comment:
Why do you assume `HasAntitoneBasis` instead of just `HasBasis`? Of course in a lot of cases you'll get an `HasAntitoneBasis` directly, but in other cases you need to do more work to get it, and I don't see why you'd want to do it?

Reply:
We already have `HasCountableBasis.isCountablyGenerated`. I'll add `HasBasis.isCountablyGenerated` as well.

Reply:
I'm aware of `HasCountableBasis.isCountablyGenerated`, but my point was that the `HasBasis` version would be a drop-in replacement for the `HasAntitoneBasis` one, since dot notation should be able to find its way through the `extends`, while you would need some extra glue to use `HasCountableBasis` directly.